client/html/blocks/menu/menu.html

```diff
@@ -1,4 +1,4 @@
-<nav class="menu page__menu" ng-class="{menu_closed: isNavHide}">
+<nav class="menu page__menu" ng-controller="MenuCtrl as MenuCtrl" ng-class="{menu_closed: MenuCtrl.GlobalData.isMenuHide}">
```

Comment:
Use some shortcut name in controller as syntax - just "menu" will be preferable.